src/vs/base/browser/ui/button/button.ts

```diff
@@ -151,6 +151,9 @@ export class Button extends Disposable {
 			this.$el.addClass('monaco-text-button');
 		}
 		this.$el.text(value);
+		//{{SQL CARBON EDIT}}
+		this.$el.attr('aria-label', value);
+		//{{END}}
```

Comment:
we have a ADS-specific button class at src/sql/base/browser/ui/button/button.ts, would it work to make this change there instead?

Reply:
I can't override the setter, will cause a compilation error.